db-client/src/main/java/com/splicemachine/db/client/am/ResultSet.java

```diff
@@ -153,7 +156,7 @@
     public boolean isAfterLast_ = false;
     public boolean isFirst_ = false;
     public boolean isLast_ = false;
-    public boolean rowsetContainsLastRow_ = false;
+    //public boolean rowsetContainsLastRow_ = false;
```

Comment:
Can you remove these lines rather than commenting them out?

Reply:
OK, for the lines in `getRef()` and `getArray()`, should I remove them, too? Not sure if we would need them later.

Reply:
Done.